src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -118,19 +123,25 @@ def run_gridscan(
     yield from set_fast_grid_scan_params(fgs_motors, parameters.grid_scan_params)
     yield from wait_for_fgs_valid(fgs_motors)
 
-    @bpp.stage_decorator([eiger])
+    @bpp.set_run_key_decorator("do_fgs")
+    @bpp.run_decorator(md={"subplan_name": "do_fgs"})
     def do_fgs():
         yield from bps.wait()  # Wait for all moves to complete
         yield from bps.kickoff(fgs_motors)
         yield from bps.complete(fgs_motors, wait=True)
 
     with TRACER.start_span("do_fgs"):
-        yield from do_fgs()
+        try:
+            yield from do_fgs()
+        except Exception as e:
+            artemis.log.LOGGER.info(e, exc_info=True)
```

Comment:
Must: Why are we logging this here? If we need to it should probably be worse than `info`, also we probably want to re-raise so as to do things like kill the RE

Reply:
wow, I really messed something up here, this was my hacky workaround for running the full plan with S03